pkg/pipeline-selector/pipeline_selector_test.go

```diff
@@ -112,9 +112,17 @@ func TestSelectPipelineForComponent(t *testing.T) {
 						Selectors: []buildappstudiov1alpha1.PipelineSelector{
 							{
 								Name: "Java spring",
-								PipelineRef: tektonapi.PipelineRef{
-									Name:   "java-spring-build-pipeline",
-									Bundle: "java-spring-bundle",
+								PipelineRef: buildappstudiov1alpha1.BackwardsCompatiblePipelineRef{
```

Comment:
Does it make sense to have a utility function that takes `Name` and `Bundle` and returns the whole structure? WDYT?

Reply:
That would be good, yeah :+1: 

Where should that function live so that it can be used both in the `controllers/` tests and in `pkg/pipeline-selector/` tests? It looks like you can't have test-only utility functions that would be usable across multiple packages :confused: 

Maybe a public function in `pkg/pipeline-selector/pipeline_selector.go`?

Reply:
It mostly used in tests, so I'd put it a test file.

Reply:
That's the thing, I defined it in `pkg/pipeline-selector/pipeline_selector_test.go`, but now I can't import it in the controller tests
![image](https://github.com/redhat-appstudio/build-service/assets/38193343/290d9147-a5a3-4e09-bbb3-3db9a7db613e)

According to Reddit, it's not really possible to share test-only utilities across packages https://www.reddit.com/r/golang/comments/jjmsoc/how_to_share_test_utilities_accros_packages/

I guess I can copy-paste the helper function in both `pkg/pipeline-selector` tests and `controllers` tests